### PR TITLE
Mark `core-js@3` as stable

### PR DESCRIPTION
**Pull request by zloirock**
https://github.com/zloirock/core-js/blob/master/docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -257,11 +257,12 @@ process.nextTick(function () {
       name: 'TypeScript + core-js',
       url: 'https://www.typescriptlang.org/',
       target_file: 'es6/compilers/typescript-core-js.html',
-      polyfills: ["node_modules/core-js/client/core.js"],
+      polyfills: ["node_modules/core-js-bundle/index.js"],
       compiler: function (code) {
         return ts.transpile(code, { downlevelIteration: true });
       }
     },
+    /*
```
Comment (ljharb) [inline build.js:265]: This should be removed?

File: build.js [hunk 2]
```diff
@@ -27,9 +27,9 @@ var environments = require('./environments');
 
 var fs = require('fs');
 var path = require('path');
-var os = require('os');
+// var os = require('os');
```
Comment (ljharb) [inline build.js:30]: Why are these commented?

File: package.json [hunk 3]
```diff
@@ -13,14 +13,13 @@
...
     "babel-core": "latest",
-    "babel-polyfill": "latest",
     "babel-preset-es2015": "latest",
     "babel-preset-es2016": "latest",
     "babel-preset-es2017": "latest",
     "babel-preset-stage-0": "latest",
     "chalk": "^1.1.3",
     "cheerio": "^0.20.0",
-    "core-js": "latest",
+    "core-js-bundle": "^3.0.0",
```
Comment (ljharb) [inline package.json:22]: if the package name is changing, shouldn’t that be communicated somehow?

File: package.json [hunk 4]
```diff
@@ -33,6 +32,7 @@
     "jstransform": "latest",
     "lodash.pickby": "^4.6.0",
     "object.assign": "^4.0.4",
+    "regenerator-runtime": "^0.13.2",
```
Comment (ljharb) [inline package.json:35]: Why is this needed at the top level?

File: build.js (hunk 1 above)
Comment (zloirock) [reply build.js:265]: Maybe, it can be fixed in the future.

File: build.js (hunk 2 above)
Comment (zloirock) [reply build.js:30]: Because it's required for Closure Compiler tests which now can't be built.

File: build.js (hunk 1 above)
Comment (ljharb) [reply build.js:265]: That seems like something that needs to be fixed separately. cc @teppeis ?

File: package.json (hunk 3 above)
Comment (zloirock) [reply package.json:22]: It's the package only with bundled version. Communicated - what do you mean? It's renamed in cases of usage.

File: package.json (hunk 4 above)
Comment (zloirock) [reply package.json:35]: For replacement of deprecated and obsolete `babel-polyfill`.

File: package.json (hunk 3 above)
Comment (ljharb) [reply package.json:22]: why do we need the bundled version here?

File: package.json (hunk 4 above)
Comment (ljharb) [reply package.json:35]: what about `@babel/polyfill`?

Comment (ljharb) [reply package.json:35]: either way everything is labeled as “with core-is”, not “with regenerator-runtime”. Should we remov this and update the test results?

File: package.json (hunk 3 above)
Comment (zloirock) [reply package.json:22]: For generation of pages with test cases? See the source code of `build.js`.

File: package.json (hunk 4 above)
Comment (zloirock) [reply package.json:35]: `@babel/polyfill` is dead. `regenerator` transform is a part of `babel`, I don't think that any additional labels required.

File: build.js (hunk 1 above)
Comment (zloirock) [reply build.js:265]: Seems since @teppeis provide data not only for ES6, he has an alternative tests runner.

Comment (teppeis) [reply build.js:265]: @ljharb @zloirock I'm using my own build scripts for closure compiler: https://github.com/teppeis/closure-compiler-es-compat-table
Because it's too slow to run tests one by one in Java, I use [nailgun](https://github.com/facebook/nailgun) to reduce the overhead of starting JVM and add some convenient features.

So it is not a problem for me to delete this closure part of build.js, but I do not know if it should be deleted for compat-table project.

**Comment by ljharb** [comment]
I don’t see any approvals on this yet.

**Comment by zloirock** [comment]
Hmm... You did not request any changes, I answered your questions, there was enough time. If you are against something - feel free to open a new PR or issue and we will discuss it.

**Comment by ljharb** [comment]
The lack of a block isn’t approval; I’ll consistently block in the future.

**Comment by zloirock** [comment]
Can you mark any points in this pull request you don't agree with?

**Comment by ljharb** [comment]
Everything i commented. For one, if the label is going to say “core-js” then regenerator-runtime and it’s results should not be included (and shouldn’t have been previously). If it’s going to include regenerator-runtime, then the label should include that. I also don’t think we should have committed the code commenting out the broken closure compiler bit - i think that should stay broken in master til it’s fixed (and should have CI that exercises it so that can’t happen again). I also think that if we’re using core-js-bundle then that might be misleading to users who would likely use core-js; i think we should use core-js itself (and do our own bundling if needed).

**Comment by zloirock** [comment]
> Everything i commented.

And I answered. You had 17 hours to say that you disagree.

> For one, if the label is going to say “core-js” then regenerator-runtime and it’s results should not be included (and shouldn’t have been previously).

Really?  As I wrote, `babel` contains `regenerator` transform, `regenerator-runtime` is a runtime for that. We haven't any labels about runtimes for other engines.

>  I also don’t think we should have committed the code commenting out the broken closure compiler bit - i think that should stay broken in master til it’s fixed (and should have CI that exercises it so that can’t happen again).

This code is broken on `master` for a long time. And it prevents adding the new result for compilers for a long time. Maybe because of this we have outdated results for almost all of them.

> I also think that if we’re using core-js-bundle then that might be misleading to users who would likely use core-js; i think we should use core-js itself (and do our own bundling if needed).

Previously, `compat-table` used bundled `core-js` version from `core-js` package. Now it's just moved to a separate package, so it's a strange issue. Feel free to rewrite tests builder if you think that it's a good idea, I haven't any time for that.
